<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AreaWindow</class>
 <widget class="QDialog" name="AreaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Площа</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Площа</string>
   </property>
  </widget>
  <widget class="QLabel" name="figureName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="firstParamLabel">
        <property name="text">
         <string>FirstParam</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="secondParamLabel">
        <property name="text">
         <string>SecondParam</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="calcBtn">
      <property name="text">
       <string>Підрахувати</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="getFromFileBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Зчитати з файлу</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
